commitizen/commands/bump.py

```diff
@@ -103,6 +103,7 @@ def __call__(self):  # noqa: C901
         is_yes: bool = self.arguments["yes"]
         increment: Optional[str] = self.arguments["increment"]
         prerelease: str = self.arguments["prerelease"]
+        devrelease: bool = self.arguments["devrelease"]
```

Comment:
Why is this a bool?

Reply:
A leftover from an earlier attempt when it was; fix incoming...

Reply:
@woile, `devrelease` should be an `Optional[int]` here, right?

Should `prerelease` be an `Optional[str]`? Comes through as `None` if `--prerelease` isn't passed—if that's correct, shall I fix that while I'm correcting this?

Reply:
Yes, it should be an `Optional[int]`, as specified [here](https://github.com/commitizen-tools/commitizen/pull/582/files#diff-6a6eecba6ea4aab85dcb0372fe57ce7306496e5a47186850be245bd732e8894eR147)